### DISCUSSION TITLE
feature request to add ability to use cloud gpu instead of local gpu

### DISCUSSION THREAD

**Discussion by GeorvityLabs** [NVlabs/instant-ngp#592] [Ideas / Feature requests]
hope someone could add colab gpu support/kaggle gpu integration to the UI , that way user could login/use API key to access colab gpu and see the results on their local machine.
Maybe an added UI element to the existing window?

**Comment by realnikolaj** [comment]
I like this idea and think implementing in the way that Jupyter lab/server works would be great. Currently trying to compile on an HPC and getting stuck at x11_init and c99 not allowed, which I believe is caused by the desktop session not running on the same node which I'm compiling on - tried compiling via terminal with X11 forwarding, and without. I know this would never be optimal because network latencies but I want to try out an idea for using instant-ngp to do confidence on actual dimension rendered and comparing to true dimensions of the environment, and have no other access to CUDA than this HPC.
